src/Core/Framework/Api/EventListener/CorsListener.php

```diff
@@ -16,6 +16,11 @@
 #[Package('core')]
 class CorsListener implements EventSubscriberInterface
 {
+
+    public function __construct(private EntityRepository $salesChannelDomainRepository)
```

Comment:
can you use plain sql? therefore when a OPTIONS request happens we don't initialize a lot services

Reply:
Updated with some improvements